elastic/kibana
# [ResponseOps][Alerting] xpack.actions.proxyUrl is not validated as a URL at startup

Resolves https://github.com/elastic/kibana/issues/113316

## Summary
Added validation for `xpack.actions.proxyUrl`. If the `proxyUrl` is not valid, kibana will log a warning.


### Checklist


### To verify

- Start kibana with the following command, make sure to substitute a bad url.
`yarn start --run-examples --xpack.actions.proxyUrl=<bad url>  `

- Verify that kibana logs the following warning
`The confguration xpack.actions.proxyUrl: <bad url> is invalid.`
- Run the above command with valid proxy url and verify that kibana does not log a warning
